src/Event/Event.php

```diff
@@ -587,7 +587,7 @@ protected function createTitleTranslatedEvent(LegacyLanguage $language, Title $t
 
     protected function createTitleUpdatedEvent(Title $title): TitleUpdated
```

Comment:
Does it make sense to change the interface of this method and change the type of `$title` to `string`?